.ci/actions/java/action.yml

```
@@ -0,0 +1,20 @@
+name: 'Java + Maven configuration'
+description: 'Java and Mavn version setup'
+inputs:
+  java-version:
+    description: "the java version"
+    default: 8
```

Kogito supports JDK >= 11
```suggestion
    default: 11
```